p2p/libp2p/netMessenger.go

```diff
@@ -373,18 +354,11 @@ func (netMes *networkMessenger) CreateTopic(name string, createChannelForTopic b
 	}
 
 	netMes.topics[name] = nil
-	topic, err := netMes.pb.Join(name)
+	subscrRequest, err := netMes.pb.Subscribe(name)
```

Comment:
This seems a big change. Does pb.Subscribe do a topic.Join and a topic.Subscribe inside? Just a clarification for me.

Reply:
Join actually creates the new topic that has the methods Subscribe and Publish. This was the major change when I have adapted the new libp2p libs. Will further investigate when the new libs will be re-integrated

Reply:
ok